Modules/posixmodule.c

```diff
@@ -10422,19 +10422,24 @@ os_lockf_impl(PyObject *module, int fd, int command, Py_off_t length)
 os.lseek -> Py_off_t
 
     fd: int
+        An open file descriptor, as returned by os.open().
     position: Py_off_t
-    how: int
+        Position, interpreted relative to 'whence'.
+    whence as how: int
+        The relative position to seek from. Valid values are:
+        - SEEK_SET: seek from the start of the file.
+        - SEEK_CUR: seek from the current file position.
+        - SEEK_END: seek from the end of the file.
     /
 
-Set the position of a file descriptor.  Return the new position.
+Set the position of a file descriptor, and return the new position.
```

Comment:
Perhaps:
```suggestion
Set and return the position of a file descriptor.
```

Reply:
I think this is good.

Reply:
OTOH, the suggested change is very explicit and clear. Perhaps it makes for less confusion.

Reply:
No strong opinion, it was an attempt to reduce some duplication -- but just a cosmetic change rather than substance.

Reply:
I'll ponder on it until tomorrow :)

Reply:
Reverted in ca27fca6e967b2cc43d8ad465bc1354515c9fb8e